.travis.yml

```diff
@@ -22,8 +22,65 @@ before_script:
   - source ci/setup_env.sh
   - ci/travis_keep_alive.sh
 
+.com.github.horizenofficial.sidechain-sdk.job-definitions:
+  - &test_docker_amd64_x-large
+    if: NOT env(SKIP_TESTS)
+    os: linux
+    dist: focal
+    vm:
+      size: x-large
+    script:
+      - bash -c "chmod +x ${TRAVIS_BUILD_DIR}/ci/run_python_tests.sh"
+      - bash -c "${TRAVIS_BUILD_DIR}/ci/run_python_tests.sh ${TEST_CMD} ${TEST_ARGS}"
+
 jobs:
+  fast_finish: true
   include:
+    - stage: Test
+      <<: *test_docker_amd64_x-large
+      env:
+        - TEST_CMD="./run_sc_tests.sh"
```

Comment:
It looks like TEST_CMD is repeating in all the tests, Perhaps, move under your `job-definition` anchor as well?

Reply:
Can't use that for only "- TEST_CMD="./run_sc_tests.sh" it has to be for whole section